search/readme.md

```diff
@@ -0,0 +1,119 @@
+# Search API
+
+## General
+Search API gives you the ability to run queries via REST (i.e. HTTP). The query language is elasticsearch Search API DSL, with certain limitations. 
```

Comment:
The Search API allows Logz.io users to run queries via REST (i.e. HTTP). The query language used is Elasticsearch Search API DSL, with certain limitations.